docs/examples/notebooks/ShapeFactor.ipynb

```diff
@@ -218,41 +230,11 @@
     }
    ],
    "source": [
-    "def plot_results(testmus, cls_obs, cls_exp, poi_tests, test_size=0.05):\n",
-    "    plt.plot(poi_tests, cls_obs, c='k')\n",
-    "    for i, c in zip(range(5), ['grey', 'grey', 'grey', 'grey', 'grey']):\n",
-    "        plt.plot(poi_tests, cls_exp[i], c=c)\n",
-    "    plt.plot(testmus, [test_size] * len(testmus), c='r')\n",
-    "    plt.ylim(0, 1)\n",
-    "\n",
-    "\n",
-    "def invert_interval(test_mus, cls_obs, cls_exp, test_size=0.05):\n",
-    "    crossing_test_stats = {'exp': [], 'obs': None}\n",
-    "    for cls_exp_sigma in cls_exp:\n",
-    "        crossing_test_stats['exp'].append(\n",
-    "            np.interp(\n",
-    "                test_size, list(reversed(cls_exp_sigma)), list(reversed(test_mus))\n",
-    "            )\n",
-    "        )\n",
-    "    crossing_test_stats['obs'] = np.interp(\n",
-    "        test_size, list(reversed(cls_obs)), list(reversed(test_mus))\n",
-    "    )\n",
-    "    return crossing_test_stats\n",
-    "\n",
-    "\n",
-    "poi_tests = np.linspace(0, 5, 61)\n",
-    "tests = [\n",
-    "    pyhf.infer.hypotest(\n",
-    "        poi_test, data, pdf, init_pars, par_bounds, return_expected_set=True\n",
-    "    )\n",
-    "    for poi_test in poi_tests\n",
-    "]\n",
-    "cls_obs = np.array([test[0] for test in tests]).flatten()\n",
-    "cls_exp = [np.array([test[1][i] for test in tests]).flatten() for i in range(5)]\n",
+    "import pyhf.contrib.viz.brazil\n",
     "\n",
-    "print('\\n')\n",
-    "plot_results(poi_tests, cls_obs, cls_exp, poi_tests)\n",
-    "invert_interval(poi_tests, cls_obs, cls_exp)"
+    "fig, ax = plt.subplots(figsize=(10, 7))\n",
+    "pyhf.contrib.viz.brazil.plot_results(ax, poi_tests, tests, test_size=0.05)\n",
+    "{'exp': exp_limits, 'obs': obs_limit}"
```

Comment:
I might go ahead and print this in a more regular fashion, instead of just having the dict get dumped out.